### PR TITLE
[new release] wayland (0.1)

### PR DESCRIPTION
**Pull request by talex5**
Pure OCaml Wayland protocol library

- Project page: <a href="https://github.com/talex5/ocaml-wayland">https://github.com/talex5/ocaml-wayland</a>
- Documentation: <a href="https://talex5.github.io/ocaml-wayland/">https://talex5.github.io/ocaml-wayland/</a>

##### CHANGES:

Initial release.

### CODE REVIEW

File: packages/wayland/wayland.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Pure OCaml Wayland protocol library"
+description:
+  "Wayland is a communications protocol intended for use between processes on a single computer. It is mainly used by graphical applications (clients) to talk to display servers, but nothing about the protocol is specific to graphics and it could be used for other things. This library can be used to write Wayland clients, servers and proxies."
+maintainer: ["talex5@gmail.com"]
+authors: ["talex5@gmail.com"]
+homepage: "https://github.com/talex5/ocaml-wayland"
+doc: "https://talex5.github.io/ocaml-wayland/"
+bug-reports: "https://github.com/talex5/ocaml-wayland/issues"
```
Comment (mseri) [inline packages/wayland/wayland.0.1/opam:9]: Can you add the license field?

Comment (talex5) [reply packages/wayland/wayland.0.1/opam:9]: Would `license: "Apache-2.0 (excluding schema files)"` be OK?

(I've also deployed your patch to CI to catch this)

Comment (mseri) [reply packages/wayland/wayland.0.1/opam:9]: Sounds good to me, thanks!
(and thanks for the deployment)

Comment (kit-ty-kate) [reply packages/wayland/wayland.0.1/opam:9]: If some of the files have a different license you can list them too, the `license` field accepts a list of license (e.g. `license: ["Apache-2.0" "MIT"]`)

**Pull request closed without merging**